tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```
@@ -555,6 +573,16 @@ Services:
     - --event-size 1024 --max-batch-size 100 --preload 1000000 --transport-type 1 --uamqp-transport --sync --warmup 30
     TestNames:
       Python: ProcessEventsBatchTest
+  - Test: send-event-batch
```

```suggestion
  - Test: send
```

Need to change this name to match name used in JS PR, and also to avoid conflict with name `send-event-batch` already used by Python.